README.rst

```diff
@@ -88,7 +89,10 @@ Before running tests, please ensure that following environment variables are set
     ==> BASIC_AUTH_PASSWORD
     ==> USER_LOGIN_EMAIL
     ==> USER_LOGIN_PASSWORD
-
+    ==> COURSE_RUN
+    ==> COURSE_DISPLAY_NAME
+    ==> COURSE_NUMBER
+    ==> COURSE_ORG
 
 To run all the tests:
 
```

Comment:
@raeeschachar  I think we should include the flaky tests and exclude flag in a note after the "How to run tests" section. 

However, I'm not sure whether the e2e_test flag exludes whitelabel tests by default or not. Need to check that up. 

as per the above discussion with @estute I can't figure out what have we agreed upon as our final choice out of the two?

Also, in another conversation on this same PR, @estute  has mentioned that we use a different paver target for running the whitelabel tests i.e **e2e_wl_test**. so in that case I guess **e2e_test** paver target will not run whitelabel tests. Thoughts?